src/Engine/Manager/SoundManager.cpp

```diff
@@ -77,7 +77,7 @@ void SoundManager::Update(float deltaTime) {
 
             float* soundBuf = new float[numSamples];
             if (sound->soundBuffer->GetSize() > sound->place + numSamples) {
-                std::memcpy(soundBuf,(sound->soundBuffer->GetBuffer() + sound->place), sizeof(float)*numSamples);
+                std::memcpy(soundBuf, (sound->soundBuffer->GetBuffer() + sound->place), sizeof(float)*numSamples);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)